test/common_quantization.py

```diff
@@ -352,30 +353,50 @@ def forward(self, x):
         return self.dequant(x)
 
 
-class SubModForFusion(torch.nn.Module):
+class SubModelForFusion(nn.Module):
     def __init__(self):
-        super(SubModForFusion, self).__init__()
-        self.conv = torch.nn.Conv2d(20, 20, 1, bias=None)
-        self.bn = torch.nn.BatchNorm2d(20)
+        super(SubModelForFusion, self).__init__()
+        self.conv = nn.Conv2d(2, 2, 1, bias=None).to(dtype=torch.float)
```

Comment:
Why do we need to convert the layer to `float`? I think by default it assumes so.

Reply:
by default it is torch.double